modules/chapter2/pages/rbac.adoc

```diff
@@ -0,0 +1,170 @@
+# Role Based Access Control (RBAC)
+:navtitle: Role Based Access Control
+
+In the previous section, you learnt how to _authenticate_ users (that is, *who* they are). In this section you will learn how to _authorize_ users to perform actions in RHDH (that is, *what* they can do in RHDH).
+
+Role-Based Access Control (RBAC) is a security concept that controls access to resources in a system, and specifies a mapping between _users_ of the system, and the _actions_ they can perform on _resources_ in the system. You define _roles_ with specific _permissions_ , and then assign the roles to users.
+
+RBAC on RHDH is built on top of the *Permissions* framework in Backstage, which defines RBAC policies in code. Rather than define policies in code, the RHDH RBAC feature allows you to define policies in a declarative fashion using a simple CSV based format.
+
+To apply RBAC in RHDH, you need to do two things:
+
+. Configure an authenticated user as a *Policy Administrator*
+. Configure policies in a CSV file and import it into RHDH
+
+## Declarative Role Based Access Control
+
+RBAC in RHDH is configured using a CSV file that follows the *casbin* (https://casbin.org) format, a popular declarative domain specific language (DSL) for access control.
+
+A RBAC policy CSV consists of a number of lines of declarations. A *policy* declaration consists of lines in the following format:
+
+[source,csv]
+----
+type, role_name, resource, action, permission
+----
+where,
+
+* *type* = should be a single letter - `p` for policy type, or `g` for role assignments to users
+* *role_name* = A custom role name defined by you, for example `admins`, `users`, `viewers`
+* *resource* = features in RHDH, for example `catalog-entity`, `policy-entity`, `scaffolder-template`, and more
```

Comment:
Should we list them all or point them to where they can get a list of all of them?

Reply:
added references to RBAC permissions markdown doc